handbook/engineering/hiring/software-engineer-code-insights.md

```diff
@@ -0,0 +1,114 @@
+   - _Note: while we do not disqualify candidates who do not have React and TypeScript experience from taking the excercise, you will be required to use these for the excercise._
+1. An **Engineer** and a **Designer** will review and grade your submission.
+1. We schedule a **1 hour follow-up call** to discuss your submission and ask any questions we might have to clarify our understanding of your technical capability and sutiability for the role.
+   - You will interview with one teammate from each of the following 2 lists
+      - **Engineer**:
+         - [Felix Becker](../../../company/team/index.md#felix-becker)
+         - [Erik Seliger](../../../company/team/index.md#erik-seliger)
+      - **Designer**:
+         - [Rob Rhyne](../../../company/team/index.md#rob-rhyne)
+         - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her)
+         - [Quinn Keast](../../../company/team/index.md#quinn-keast-he-him)
+1. We schedule the following additional interviews, in no particular order, across multiple days:
+   - 1 hour **Technical experience:** We ask you about your past work and accomplishments in depth, how you worked with others, decisions you made, and what you'd do differently today.
+      - You will interview with two teammates from the following list
+        - [Felix Becker](../../../company/team/index.md#felix-becker)
+        - [Marek Zaluski](../../../company/team/index.md#marek-zaluski)
+        - [TJ Kandala](../../../company/team/index.md#tharuntej-kandala-he-him)
+        - [Erik Seliger](../../../company/team/index.md#erik-seliger)
+   - 1 hour **Team collaboration:** We ask you about how you work and communicate in a team setting, how you handle tricky situations, decisions you made, and what you'd do differently today.
+      - You will interview with one teammate from each of the following 2 lists
+         - **Engineer**:
+            - [TJ Kandala](../../../company/team/index.md#tharuntej-kandala-he-him)
+            - [Loïc Guychard](../../../company/team/index.md#lo%c3%afc-guychard)
+            - [Marek Zaluski](../../../company/team/index.md#marek-zaluski)
+         - **Non-engineer**:
+            - [Rob Rhyne](../../../company/team/index.md#rob-rhyne)
+            - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her)
+            - [Quinn Keast](../../../company/team/index.md#quinn-keast-he-him)
+            - [María Craig](../../../company/team/index.md#mar%c3%ada-craig-she-her)
+            - [Eric Brody-Moore](../../../company/team/index.md#eric-brody-moore)
+            - [Jonah Dueck](../../../company/team/index.md#jonah-dueck-he-him)
+            - [Josh Saunders](../../../company/team/index.md#josh-saunders)
+            - [Nick McMillen](../../../company/team/index.md#nick-mcmillen-he-him)
+            - [Christine Lovett](../../../company/team/index.md#christine-lovett-she-her)
```

Comment:
I don't think it is a good thing to have such a massive interviewer list. Also, the product team is constrained, and I think it makes sense for only product/design that are on the team to interview candidates (by default).

Here is my suggestion:
```suggestion
      - You will interview with one person from each of these lists:
         - **Product/design**
            - [Joel Kwartler](../../../company/team/index.md#joel-kwartler-he-him)
            - [Alicja Suska](../../../company/team/index.md#alicja-suska-she-her)
            - Can backfill with any product manager, designer, or engineering manager.
         - **Engineering management**
            - [Loïc Guychard](../../../company/team/index.md#lo%c3%afc-guychard)
            - [Marek Zaluski](../../../company/team/index.md#marek-zaluski)
            - [TJ Kandala](../../../company/team/index.md#tharuntej-kandala-he-him)
            -  Can backfill with any engineering manager.
```

Reply:
I had copy-pasted this from the frontend JD per Jean's ask above, but I think you're absolutely right. Would also add Jean himself to the list of EMs here since I'll be doing the HM screen.

Reply:
This would however remove all people from CE and bizops, which I think were added not too long ago. Once we move the the frontend JD to individual JDs and do the same there, they would effectively not be in the panel in anymore.  Is that intentional?